core/src/main/java/nl/nn/adapterframework/core/PipeLine.java

```diff
@@ -79,11 +76,20 @@
  * If the processing of the message concluded without exceptions and the status of the transaction is
  * STATUS_ACTIVE (i.e. normal) the transaction will be committed. Otherwise it will be rolled back,
  * or marked for roll back by the calling party.
-
- * </p>
- * 
+ *
  * @author  Johan Verrips
  */
+/*
+ * Processor and keeper of a line of {@link IPipe Pipes}.
+ * <br/>
+ * Pipelines also generate statics information per Pipe and keep forwards, that are registered
+ * at individual pipes during the configure phase.
```

Comment:
```suggestion
 * Pipelines also maintain statics information per Pipe.
```

Het registreren bij de pipes in de configuratiefase is een implementatie detail dat niet van belang is voor Frankontwikkelaars

Reply:
Dit is geen JavaDoc commentaar meer maar gewoon commentaar. De Frank ontwikkelaar krijgt het niet te zien. Ik heb bestaande JavaDoc commentaren die ik niet geschikt vind voor Frank ontwikkelaars veranderd in gewoon commentaar zodat jullie het als contributors nog hebben.